tests/transactions/tests.py

```diff
@@ -218,6 +219,17 @@ def test_prevent_rollback(self):
             transaction.savepoint_rollback(sid)
         self.assertQuerysetEqual(Reporter.objects.all(), ['<Reporter: Tintin>'])
 
+    def test_rollback_on_keyboardinterrupt(self):
+        try:
+            with transaction.atomic():
+                Reporter.objects.create(first_name='Haddock')
```

Comment:
That's actually the last name of a character in the comic these tests are based upon :-)